### PR TITLE
Added issue templates for bugs and feature requests

### PR DESCRIPTION
**Pull request by indigozero**

### CODE REVIEW

**Comment by Kryptos-FR** [comment]
There is already one for bug report though: https://github.com/xenko3d/xenko/blob/master/docs/ISSUE_TEMPLATE.md

**Comment by xen2** [comment]
Yes that old docs/ISSUE_TEMPLATE.md should be removed.

**Comment by indigozero** [comment]
Probably wouldn't be too bad of an Idea, that way I wouldn't have a blank description ;)

**Comment by indigozero** [comment]
Probably would be a good idea for Pull Requests, that is.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```
Comment (xen2) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Maybe we should add bug as default label and add another (almost empty) template for "Questions" and/or "Others"?

Comment (indigozero) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I like the idea of a template for "'Questions' and/or 'Others'". Will automatically adding "bug" as default confuse us as to which issues have been verified as and actual bug as opposed to something like user error?

Comment (indigozero) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I thought that "Questions and/or Comments" sounded better, but I'd be open to changing it. I also have the template set to auto label with 'question'.

Comment (xen2) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I would say it's OK if bug is set by default, we can work the opposite way (close issue and/or remove bug tag if it wasn't a bug).

File: .github/ISSUE_TEMPLATE/questions_request.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+name: Questions and/or Comment
+about: Have a questions that needs to be answered?
```
Comment (Kryptos-FR) [inline .github/ISSUE_TEMPLATE/questions_request.md]: We might need to inform the user that general purpose questions can also be answered on our forums and discord community. So they could first try there.

Especially questions such as "how to create XXX?" or "how do I use YYY?"

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (indigozero) [reply .github/ISSUE_TEMPLATE/bug_report.md]: bug label added

File: .github/ISSUE_TEMPLATE/questions_request.md (hunk 2 above)
Comment (indigozero) [reply .github/ISSUE_TEMPLATE/questions_request.md]: @Kryptos-FR Let me know if my latest commit works for you as a solutions and if any rewording

**Comment by indigozero** [comment]
@xen2 @Kryptos-FR I want to make sure the [pull_request_template](https://github.com/indigozero/xenko/commit/ee58ffecf0b53afd1d80d2c3569da1fae3aa47a5) works for you guys also.

**Comment by Kryptos-FR** [comment]
@indigozero LGTM

File: .github/ISSUE_TEMPLATE/questions_request.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
```
Comment (xen2) [inline .github/ISSUE_TEMPLATE/questions_request.md]: Small typo: questions => question

Comment (xen2) [reply .github/ISSUE_TEMPLATE/questions_request.md]: Never mind I can edit it inline, I will just do that.

**Comment by xen2** [comment]
LGTM, merging! Thanks!

**Pull request merged**